node-tests/blueprints/component-test.js

```diff
@@ -270,6 +271,52 @@ describe('Blueprint: component', function () {
         }
       );
     });
+
+    it('component foo --strict', function () {
+      return emberGenerateDestroy(['component', 'foo', '--strict'], (_file) => {
+        expect(_file('app/components/foo.gjs')).to.equal(
+          fixture('component/template-only-component.gjs')
```

Comment:
This one still fails because detyping the .gts version has a different output. We should probably use a .gjs without detyping for this scenario.